data/json/items/gun/38.json

```diff
@@ -201,19 +201,17 @@
   {
     "id": "sw_619",
     "copy-from": "pistol_revolver",
-    "//": "Tileset whitelist for revolvers.",
     "type": "GUN",
-    "name": { "str_sp": "S&W 619" },
-    "description": "A seven-round .357 revolver sold by Smith & Wesson.  Can also fire .38 Special.  It features a fixed rear sight and a reinforced frame.",
+    "name": { "str_sp": "S&W 686 Plus" },
+    "description": "Introduced in the 90's, the Smith & Wesson's model 686 'Plus' features a robust stainless steel frame, an adjustable front sight and a seventh shot that's sure to save your life.",
     "ascii_picture": "sw_610",
-    "weight": "1074 g",
-    "volume": "608 ml",
-    "longest_side": "278 mm",
-    "barrel_length": "140 mm",
-    "//2": "100mm + 40mm max OAL for 357mag. Revolvers just built different.",
-    "price": "620 USD",
+    "weight": "1280 g",
+    "volume": "760 ml",
+    "longest_side": "305 mm",
+    "//": "barrel length (152mm) + cylinder length (40mm)",
+    "barrel_length": "192  mm",
+    "price": "980 USD",
     "price_postapoc": "12 USD 50 cent",
-    "to_hit": -2,
```

Comment:
I thought that according to #74597 we were supposed to stop manually giving items that aren't effective in melee `to-hit` fields.

Is that a misunderstanding? Wouldn't using `"to_hit": { "grip": "bad" }` be basically the same?

Reply:
As I understand it, that PR removes the old, subjective, numeric to hit values and relies upon swingable items to utilise defined descriptive to hit values. Not required for something like a pair of jeans, hitting somebody with it would do nothing anyhow, but I can clock somebody pretty well with a steel-framed revolver.

Reply:
Got it, I'll apply the changes.